combat-trainer.lic

```diff
@@ -1102,7 +1102,19 @@ class ManipulateProcess
 
   def manipulate
     bput('manip stop all', 'You relax your will', 'But you aren')
-    bput("manipulate friendship #{@filtered_npcs.first}",  'You attempt to empathically manipulate', 'You strain')
+    @manipulate_count = 0
+
+    @filtered_npcs.each_with_index { |npc, index|
+      if @manipulate_count >= @threshold
```

Comment:
You can make this a one-liner: `break if @manipulate_count >= @threshold` (or `break unless @manipulate_count < @threshold` if you think that reads better).